server/controllers/finance/reports/collection_capacity/index.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * @overview ./finance/reports/collection_capacity/
+ */
+
+const _ = require('lodash');
+const moment = require('moment');
+
+const ReportManager = require('../../../../lib/ReportManager');
+const db = require('../../../../lib/db');
+
+module.exports.report = report;
+
+// path to the template to render
+const TEMPLATE = './server/controllers/finance/reports/collection_capacity/report.handlebars';
+
+const DEFAULT_OPTIONS = {
+  orientation : 'portrait',
+};
+
+/**
+ * @method report
+ *
+ * @description
+ * The HTTP interface which actually creates the report.
+ */
+async function report(req, res, next) {
+  try {
+    const qs = _.extend(req.query, DEFAULT_OPTIONS);
+    const { dateFrom, dateTo, accountId } = req.query;
```

Comment:
This needs to be removed.  The report is currently broken because we can't select an `accountId`.